src/tox/_pytestplugin.py

```diff
@@ -404,7 +404,7 @@ def mock_venv(monkeypatch):
     and cannot install any packages. """
 
     # first ensure we have a clean python path
-    monkeypatch.delenv("PYTHONPATH", raising=False)
+    monkeypatch.delenv(str("PYTHONPATH"), raising=False)
```

Comment:
@nicoddemus @RonnyPfannschmidt @asottile slight bug here inside pytest monkeypatch. The key passed in here is used to re-insert the value at teardown, instead of the key that was there. Because during the ``get`` unicode keys degrade to bytes keys this can mean that by invoking this operation we transform bytes key to unicode inside ``os.environ`` when the key passed into delenv is unicode. 

In such cases, python 2 Windows will refuse to run subprocesses leaving the user to wonder who and where and how it set non bytes value into ``os.environ``.

Reply:
Thanks for the ping: https://github.com/pytest-dev/pytest/pull/4056